From 018f2969b72158534972befe1a0378a9cd5c2bc4 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 17 May 2022 07:51:50 +0000
Subject: [PATCH] GCC: fix ambiguous references in blink::FrameLoadRequest

Add namespace to avoid confusion.

Bug: 819294
Change-Id: Ia60b4cf7a6d17ba5cd386d3fc57cc59222fcef71
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3650536
Reviewed-by: Mike West <mkwst@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1004139}
---
 third_party/blink/renderer/core/loader/frame_load_request.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/core/loader/frame_load_request.h b/third_party/blink/renderer/core/loader/frame_load_request.h
index 444b25c831245..32e3480fd395b 100644
--- a/third_party/blink/renderer/core/loader/frame_load_request.h
+++ b/third_party/blink/renderer/core/loader/frame_load_request.h
@@ -179,7 +179,9 @@ struct CORE_EXPORT FrameLoadRequest {
     impression_ = impression;
   }
 
-  const absl::optional<Impression>& Impression() const { return impression_; }
+  const absl::optional<blink::Impression>& Impression() const {
+    return impression_;
+  }
 
   bool CanDisplay(const KURL&) const;
 
-- 
2.35.3

